### PR TITLE
[develop] backgroundThrottling should be false by default

### PR DESCRIPTION
**Pull request by HarsimranSingh**
@StevenEBarbaro

### CODE REVIEW

File: src/browser/convert_options.js [hunk 1]
```diff
@@ -266,10 +267,6 @@ module.exports = {
             newOptions.preload = options.preload;
         }
 
-        if (options.hasOwnProperty('backgroundThrottling')) {
-            newOptions.backgroundThrottling = options.backgroundThrottling;
-        }
```
Comment (whyn07m3) [inline src/browser/convert_options.js]: I believe this needs to be kept here, in addition to the added default

Comment (HarsimranSingh) [inline src/browser/convert_options.js]: If it's a part of the base options then this code isn't needed. Adding window options like this actually should be discouraged and only used for options that are used internally, like notifications passing around the notification text [here](https://github.com/HadoukenIO/core/blob/develop/src/browser/convert_options.js#L253).

**Comment by rdepena** [comment]
👍

**Comment by HarsimranSingh** [comment]
Will update with tests

**Comment by HarsimranSingh** [comment]
[Clean Test Run](https://testing-dashboard.openfin.co/#/app/sessions/api/completed/591c58ff5114ac10ca6db3dd)

File: src/browser/convert_options.js (hunk 1 above)
Comment (whyn07m3) [inline src/browser/convert_options.js]: @HarsimranSingh Got it, didn't know it was not an option, just the default

**Pull request merged**